Assets/VRM10/Runtime/Components/Expression/ExpressionKey.cs

```diff
@@ -145,19 +146,24 @@ public static ExpressionKey CreateFromPreset(ExpressionPreset preset)
 
         public override string ToString()
         {
-            return _id.Replace(UnknownPresetPrefix, "");
+            return Name;
         }
 
         public bool Equals(ExpressionKey other)
         {
-            return _id == other._id;
+            // Early pruning
+            if (_hashCode != other._hashCode) return false;
+
+            if (Preset != other.Preset) return false;
+            if (Preset != ExpressionPreset.custom) return true;
+            return Name == other.Name;
```

Comment:
Fast comparison by early pruning via `_hashCode`
and
Accurate equality comparison when `_hashCode`s are collided.